elasticdl_preprocessing/tests/concatenate_with_offset_test.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from elasticdl_preprocessing.layers.concatenate_with_offset import (
+    ConcatenateWithOffset,
+)
+
+
+class ConcatenateWithOffsetTest(unittest.TestCase):
+    def test_concatenate_with_offset(self):
+        tensor_1 = tf.constant([[1], [1], [1]])
+        tensor_2 = tf.constant([[2], [2], [2]])
+        offsets = [0, 10]
+        concat_layer = ConcatenateWithOffset(offsets=offsets, axis=1)
+
+        output = concat_layer([tensor_1, tensor_2])
+        expected_out = np.array([[1, 12], [1, 12], [1, 12]])
+        self.assertTrue(np.array_equal(output.numpy(), expected_out))
+
+        ragged_tensor_1 = tf.ragged.constant([[1], [], [1]])
+        ragged_tensor_2 = tf.ragged.constant([[2], [2], []])
+        output = concat_layer([ragged_tensor_1, ragged_tensor_2])
+        expected_out = np.array([1, 12, 12, 1])
+        self.assertTrue(np.array_equal(output.values.numpy(), expected_out))
```

Comment:
For the RaggedTensor，we need not only check the values but also the row_splits and the shape

Reply:
Yes, I will use the `sparse_tensor_equal` and `ragged_tensor_equal` in test_utils.py to check.